.travis.yml

```diff
@@ -0,0 +1,59 @@
+sudo: false
+dist: trusty
+
+language: cpp
+
+compiler: clang
+
+git:
+  depth: 2
+  quiet: true
+
+cache:
+  timeout: 1000
+  directories:
+  - $HOME/llvm
+  - $HOME/build
+
+env:
+  global:
+    - TIMEOUT=2700
+
+before_script:
+    - if [[ -d $HOME/llvm/tools/clang ]]; then
+        cd $HOME/llvm && git pull;
+        cd $HOME/llvm/tools/clang && git pull;
+      else
+        git clone --depth=2 https://github.com/llvm-mirror/llvm.git $HOME/llvm -b release_70; 
+        git clone --depth=2 https://github.com/llvm-mirror/clang.git $HOME/llvm/tools/clang -b release_70;        
+        mkdir -p $HOME/llvm/tools/clang/tools/extra;
+        mkdir -p $HOME/build;
+      fi
+    - rsync -az ${TRAVIS_BUILD_DIR}/ $HOME/llvm/tools/clang/tools/extra/
+    - export TRAVIS_BUILD_DIR=$HOME/llvm/tools/clang/tools/extra/
```

Comment:
is it a good idea to modify TRAVIS_BUILD_DIR? Maybe better use a different variable.

Reply:
Done